.github/workflows/build_cmake.yaml

```
@@ -81,6 +81,12 @@ jobs:
             sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-14 100
           fi
 
+      - name: Setup MSVC dev command prompt
+        if: ${{ startsWith(matrix.name, 'Windows MSVC') }}
+        uses: TheMrMilchmann/setup-msvc-dev@v3.0.1
+        with:
+          arch: ${{ contais(matrix.name, 'x86') && 'x86' || 'x64' }}
```

actionlint (1.7.7)

undefined function "contais". available functions are "always", "cancelled", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)